### ISSUE TITLE
Cannot change Notifications styleClass

### ISSUE THREAD

**Issue by wise-coders** [controlsfx/controlsfx#1420]
The Notifications has only one public method
```
    public Notifications darkStyle() {
        styleClass.add(STYLE_CLASS_DARK);
        return this;
    }
```
but styleClass is private, and cannot be changed. I want to make the text inside a notification red, and I cannot do this as the styleClass is private. Can you please add a method getStyleClass() to it?

**Comment by RationalityFrontline** [comment]
I've made a pr that would solve this issue: #1424 . See #1423 for usage example.

**Comment by abhinayagarwal** [comment]
Fixed with #1424
